pgxn/neon/communicator.c

```diff
@@ -806,6 +855,15 @@ prefetch_read(PrefetchRequest *slot)
 	old = MemoryContextSwitchTo(MyPState->errctx);
 	response = (NeonResponse *) page_server->receive(shard_no);
 	MemoryContextSwitchTo(old);
+
+	/* Update min in-flight prefetch reqwuest LSN */
```

Comment:
This looks like duplicated code from line 529?

Reply:
Extracted in `update_min_prefetch_lsn` function